sdk/keyvault/keyvault-common/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@azure/keyvault-common",
+  "sideEffects": false,
+  "private": true,
+  "author": "Microsoft Corporation",
+  "version": "4.1.0-preview.2",
```

Comment:
I assume you picked this version to match the other keyvault libraries?

Reply:
It's a oversight! Though I'm thinking on making this 1.0.0 and not update it, like how we do with the recorder: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/test-utils/recorder